hw/hw07/vitya199909/Task1.py

```diff
@@ -0,0 +1,17 @@
+def max_of_two_numbers(a, b):
```

Comment:
гарна робота,
я б порадила додати перевірку чи аргументи числа

```python
def get_max(num1, num2):
    """
    Returns the larger of two numbers if both are numeric (int or float).
    Otherwise, returns an error message.
    """
    if not isinstance(num1, (int, float)):
        return "The first argument is not a number."
    if not isinstance(num2, (int, float)):
        return "The second argument is not a number."

    return num1 if num1 > num2 else num2